lib/ohai/plugins/linux/platform.rb

```diff
@@ -87,10 +87,12 @@ def get_redhatish_version(contents)
       platform "arch"
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 2.6.32-ARCH
+      platform_version `uname -r`.strip
```

Comment:
Should we be using shell_out instead of backticks here? @lamont-granquist @danielsdeleo